cycle_submit.py

```diff
@@ -273,16 +279,20 @@ def main():
 	else:
 		password = options.password
 
+        if options.hostname.startswith('http'):
+            hostname = options.hostname
+        else:
+            hostname = 'http://' + options.hostname
```

Comment:
Building up a "url" variable might be better than making a local "hostname" variable that is not a hostname, and also not the same as the value in "options.hostname".